react-native/src/blueTheme.ts

```diff
@@ -6,83 +6,53 @@ All rights reserved.
 This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
 **/
 
-import { blue, red, gray, lightBlue, white } from '@pxblue/colors';
-import { TextStyle } from 'react-native';
+import { Platform } from 'react-native';
+import { configureFonts, DefaultTheme, Theme } from 'react-native-paper';
+import { blue, red, gray, black, lightBlue, white } from '@pxblue/colors';
 
-type Font = {
-    fontFamily: string;
-    fontWeight: TextStyle['fontWeight'];
+const fallbackFont = Platform.OS === 'ios' ? 'System' : 'Roboto';
+type FontWeight = 'normal' | 'bold' | '100' | '200' | '300' | '400' | '500' | '600' | '700' | '800' | '900';
+
+const defaultFontConfig = {
+    regular: {
+        fontFamily: 'OpenSans-Regular' || fallbackFont,
```

Comment:
I am a bit confused, does this actually do the fallback check? Doesn't the `||` just evaluate it as a string literal, instead of doing the CSS thing?

Reply:
I borrowed this from the Shiverware implementation, but you are correct, this does not actually do any font fallback. React Native only allows you to specify a single font family. On Android, if that font family is not found, it will automatically fall back to the system default. On iOS, if it is not found, the app will throw an exception.

I will remove the fallback stuff from here since it doesn't have an effect.